webapp/src/libs/services/BaseService.ts

```diff
@@ -12,21 +12,26 @@ interface ServiceRequest {
 
 const noResponseBodyStatusCodes = [202, 204];
 
+export const BackendServiceUrl = process.env.REACT_APP_BACKEND_URI ?? window.origin;
```

Comment:
Wondering whether we should check for empty strings in addition to null ones...

Reply:
`checkEnv.ts` is currently handling this for us, but in my next PR that file will be going away. 

so now instead of checking for missing variables on start-up, we will first try to fetch the auth info with the URL that we have, and if that fails then we can point out that the `REACT_APP_BACKEND_URI` is empty if that is the case.